gadgets/trace_dns/gadget.yaml

```diff
@@ -69,22 +87,28 @@ datasources:
           columns.width: "2"
       qr_raw:
         annotations:
+          description: Raw numeric qr value (0=Q, 1=R)
+          value.one-of: "0, 1"
           columns.hidden: "true"
       qtype:
         annotations:
           description: Query type
           value.one-of: "A, NS, CNAME, SOA, PTR, MX, TXT, AAAA, SRV, OPT, WKS, HINFO, MINFO, AXFR, ALL"
       qtype_raw:
         annotations:
+          description: Raw numeric qtype value (1=A, 2=NS, 5=CNAME, 6=SOA, 12=PTR, 15=MX, 16=TXT, 28=AAAA, 33=SRV, 41=OPT, 44=WKS, 13=HINFO, 14=MINFO, 252=AXFR, 255=ALL)
```

Comment:
nit: Should we really have this for `_raw` fields? Like I want to understand if there is a use-case for it because normally I would recommend users to just use the non raw fields?

Reply:
Yes, we will always recommend users to use the non-raw fields, but we still need to tell them what those numbers are.